docs/highlighters/regexp.md

```diff
@@ -12,11 +12,32 @@ To use this highlighter, associate regular expressions with styles in the
 
 ```zsh
 typeset -A ZSH_HIGHLIGHT_REGEXP
-ZSH_HIGHLIGHT_REGEXP+=('\bsudo\b' fg=123,bold)
+ZSH_HIGHLIGHT_REGEXP+=('^rm .*' fg="red",bold)
 ```
 
-This will highlight "sudo" only as a complete word, i.e., "sudo cmd", but not
-"sudoedit"
+This will highlight the whole line starting with `rm` command (for all
+operating systems, in contrast to the below example).
+
+Some regular expressions are [subject to the host platform][MAN_ZSH_REGEX], 
+especially the kernel. To highlight `sudo` only as a complete word, i.e., 
```

Comment:
Since the zsh/regex module uses `regexec`, the dependency is on the platform's libc not the kernel. (e.g. Debian/kFreeBSD which appears to have used glibc would support `\<` not `[[:<:]]`). I'd replace "especially the kernel" with "in particular the libc" or similar.

Reply:
Thank you for correction. I'll change this part.